docs/source/guide/start.md

```diff
@@ -10,22 +10,125 @@ After you install Label Studio, start the server to start using it.
+Or, set the following environment variable:
+```
+LABEL_STUDIO_INTERNAL_HOST = localhost
+```
 
-For example, if you import data while labeling is being performed, labeling tasks can take more than 10 seconds to load and annotations can take more than 10 seconds to perform. If you want to label more than 100,000 tasks with 5 or more concurrent users, consider using PostgreSQL or another database with Label Studio. 
+## Run Label Studio on localhost with a different port
+By default, Label Studio runs on port 8080. If that port is already in use or if you want to specify a different port, start Label Studio with the following command:
+```bash
+label-studio start --port <port>
+```
 
-## Command line arguments for starting Label Studio
+For example, start Label Studio on port 9001:
+```bash
+label-studio start --port 9001
+```
 
-You can specify project config, machine learning backend and other options using the command line interface. Run `label-studio start --help` to see all available options. Many command line arguments are deprecated and removed in version 1.0.0. 
+Or, set the following environment variable:
+```
+LABEL_STUDIO_PORT = 9001
+```
+
+## Run Label Studio on Docker with a different port
+
+To run Label Studio on Docker with a port other than the default of 8080, use the port argument when starting Label Studio on Docker. For example, to start Label Studio in a Docker container accessible with port 9001, run the following: 
+```bash
+docker run -it -p 9001:8080 -v `pwd`/mydata:/root/.local/share/label-studio/ heartexlabs/label-studio:latest label-studio
+```
 
-* The Label Studio web server always uses the `0.0.0.0` address to start. If you need to change it to `localhost`, set `--internal-host` from console arguments to `localhost` and the web server starts at `localhost`.
+Or, if you're using Docker Compose, update the `docker-compose.yml` file that you're using to expose a different port for the app. For example, this portion of the `docker-compose.yml` file exposes port 9001 instead of port 8080 for running Label Studio:
+```
+...
+app:
+  build: ./
+  expose:
+    - "9001"
+...
+```
+
+## Run Label Studio with HTTPS
+To run Label Studio with HTTPS and access the web server using HTTPS in the browser, specify a certificate and private key when starting Label Studio. 
 
-* You can use the `--port` or `PORT` environment var to set port for Label Studio web server. 
+You can start Label Studio with the following command:
+```bash
+label-studio start --cert <certificate.pem> --key <keyfile.pem>
+```
 
+Or, set the following environment variables:
+```
+LABEL_STUDIO_CERT_FILE = <certificate.pem>
+LABEL_STUDIO_KEY_FILE =  <keyfile.pem>
+```
+
+The certificate and private key files must both be provided as PEM files. 
+
+## Run Label Studio on the cloud using Heroku
+To run Label Studio on the cloud using Heroku, specify an environment variable so that Label Studio loads. 
+
+```
+LABEL_STUDIO_HOSTNAME
```

Comment:
I believe this one should be `LABEL_STUDIO_HOST`

Reply:
Yeah this is copied from a Slack thread with Max but I was also guessing it should be LABEL_STUDIO_HOST